tutorials/cloud/aws.md

```diff
@@ -0,0 +1,137 @@
+# Running NoisePy on AWS
+
+
+## AWS accounts and roles.
+Make sure you have an account on AWS. AWS requires particular credentials to connect.
+
+Noisepy uses S3/Cloudstore to store the cross correlations and stacked data. For this step, it is important that your **user/role** and the **bucket** have the appropriate permissions for users to read/write into the bucket.
+
+In the browser, please add the following policy to the bucket:
+```
+{
+    "Version": "2012-10-17",
+    "Id": "Policy1674832359797",
+    "Statement": [
+        {
+            "Sid": "Stmt1674832357905",
+            "Effect": "Allow",
+            "Principal": {
+                "AWS": "arn:aws:iam::<YOUR-ACCOUNT-ID>:role/<YOURROLE>"
+            },
+            "Action": "s3:*",
+            "Resource": "arn:aws:s3:::<BUCKET-NAME>/*"
+        }
+    ]
+}
+```
+In order to check whether the user can read/write in the bucket, we recommend testing from local:
+```
+aws s3 ls s3://<BUCKET-NAME>
+```
+Add a temporary file to make sure you have the credentials to add to the bucket
+```
+aws s3 cp temp s3://<BUCKET-NAME>
+```
+
+If this step works, and if your role and user account are attached to the bucket policy, the rest of the AWS noisepy tutorial should work.
+
+## Create an EC2 instance
+
+- Log into your AWS account and go into the EC2 Dashboard
+- Click on Launch Instance
+- Select the AWS Linux OS Image
+- Create a new Key pair (RSA)
+- You can use most defaults but we recomment changing `Network Settings --> Allow SSH traffic from` to `My IP`
+
+More information about getting on the cloud in the [SCOPED HPS Book](https://seisscoped.org/HPS/softhardware/AWS_101.html).
+
+## SSH into your instance
+
+Make your private key file only readable by you (assuming it's named/downloaded to `~/Downloads/ec2.pem`). Go to your instance's summary page and copy the `Public IPv4 DNS`
+
+```
+cd ~/Downloads
+chmod 400 ec2.pem
+ssh -i ec2.pem ec2-user@<public dns name>
+```
+
+## Setup NoisePy
+
+This tutorial focuses on small, toy problems to be ran on notebooks or simple CLI. We include jupyter notebook instructions to explore the data and results.
+
+
+
+Inside your EC2 instance SSH session, run:
+
+```
+sudo yum install git
+wget https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh
+chmod +x Miniconda3-latest-Linux-x86_64.sh
+./Miniconda3-latest-Linux-x86_64.sh -b -p $HOME/miniconda
+./miniconda/bin/conda init bash
+bash
+source .bashrc
+conda create -n noisepy python==3.10
+conda activate noisepy
+git clone https://github.com/mdenolle/NoisePy
+cd NoisePy
+pip install ipykernel jupyter
+pip install noisepy-seis .
```

Comment:
The `.` would install the git-cloned code as package and would override the installation of the published `noisepy-seis` from PyPI

```suggestion
pip install noisepy-seis
```